recipes/llvm/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  10.0.0:
+    url: https://github.com/llvm/llvm-project/archive/llvmorg-10.0.0.zip
```

Comment:
Prefer `https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/llvm-project-10.0.0.tar.xz` which is 80Mb in size compared to the zip archive which is ~150Mb.

Also it may be worth updating to 10.0.1 which contains a number of bugfixes.